packages/javascript/bh-shared-ui/src/views/ZoneManagement/Details/DetailsList.test.tsx

```diff
@@ -104,6 +124,65 @@ describe('List', async () => {
         expect(screen.queryByText('app-icon-sort-empty')).not.toBeInTheDocument();
     });
 
+    it('does not render tier icon tooltip when multi tier analysis is disabled', async () => {
+        const configRes = {
+            data: [
+                {
+                    key: ConfigurationKey.Tiering,
+                    value: { multi_tier_analysis_enabled: false, tier_limit: 3, label_limit: 10 },
```

Comment:
Not sure what is happening here when I run this locally. My expectation was that if I change the `multi_tier_analysis_enabled: false` to `true` the test would fail, but it still passes. Can you check that out on your side and/or let me know if I'm misinterpreting this.